toolkit/tools/imagegen/configuration/systemconfig.go

```diff
@@ -30,6 +30,7 @@ type SystemConfig struct {
 	Encryption         RootEncryption      `json:"Encryption"`
 	RemoveRpmDb        bool                `json:"RemoveRpmDb"`
 	ReadOnlyVerityRoot ReadOnlyVerityRoot  `json:"ReadOnlyVerityRoot"`
+	HidepidDisable     bool                `json:"HidepidDisable"`
```

Comment:
Please also extend the config tests now to include the new field.

Reply:
Added to the large test structures